src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/TwigEnvironmentPass.php

```diff
@@ -28,16 +28,26 @@ public function process(ContainerBuilder $container)
             return;
         }
 
+        $prioritizedLoaders = array();
+
+        foreach ($container->findTaggedServiceIds('twig.extension', true) as $id => $attributes) {
+            $priority = isset($attributes[0]['priority']) ? $attributes[0]['priority'] : 0;
+            $prioritizedLoaders[$priority][] = $id;
+        }
```

Comment:
Could this be replaced by using the [`PriorityTaggedServiceTrait`](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/DependencyInjection/Compiler/PriorityTaggedServiceTrait.php)?

Reply:
@iltar done :)